Source/Rio.Web/src/app/pages/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -146,10 +146,10 @@ export class ManagerDashboardComponent implements OnInit, OnDestroy {
         // unlike in ParcelListComponent, the grid is not initialized until the following subscription resolves, because its container element is conditioned on parcels
         // so, we don't need to manually reset the column defs as we did in that case.
 
-        this.parcelService.getParcelsWithLandOwners(this.waterYearToDisplay.Year).subscribe(parcels=>{
-          this.parcels = parcels;
-          this.updateAnnualData();
-        });
+        this.updateAnnualData();
+        // this.parcelService.getParcelsWithLandOwners(this.waterYearToDisplay.Year).subscribe(parcels=>{
+        //   this.parcels = parcels;
```

Comment:
please remove the commented out code; no need to have it linger since we have source control

Reply:
Just removed & pushed